test/test_torch_distributed_bucketed_all_reduce_xla_backend.py

```diff
@@ -0,0 +1,35 @@
+import os
+import sys
+import torch
+import torch_xla
+import torch_xla.core.xla_model as xm
+import torch_xla.distributed.xla_multiprocessing as xmp
+import torch_xla.distributed.xla_backend
+import torch.distributed as dist
+
+
```

Comment:
Do you need to set the ALLREDUCE_GRADIENTS_BUCKET_SIZE_MB envvar for this test?

Reply:
Also, do you need to add this test to run_tests.sh?

Reply:
yea this test is not being run in CI

Reply:
we don't need the env flag, the test runs the bucketized version directly